docker/nginx/conf.d/default.conf

```diff
@@ -40,10 +40,10 @@ server {
   # SSL configuration according to best practices from
   # https://mozilla.github.io/server-side-tls/ssl-config-generator/
   ssl on;
-  # The provided certificate (test.crt) and private key (test.key) is only for
-  # testing and must never be used in production environment.
-  ssl_certificate /etc/ssl/certs/test.crt;
-  ssl_certificate_key /etc/ssl/private/test.key;
+  # The provided certificate (test.crt) and private key (test.key) specified
+  # in the .env.example is only for testing and must never be used in production environment.
+  ssl_certificate /etc/ssl/certs/nginx_cert.crt;
```

Comment:
Shouldn't you change also the file names?

Reply:
The certificate file names are specified in the `.env` file with the variables `NGINX_KEY` and `NGINX_CERT`, then are passed from the `docker-services.yml` to the nginx `Dockerfile` as `$NGINX_KEY` and `$NGINX_CERT` [(here)](https://github.com/open-education-polito/fare-platform/pull/181/files/0efdfd99829943c1cfd11ccafe41185713bb776b#diff-6817dae23246bdc9dc48fa148e00bd6e280e65975614bf06bd86030e315ee479R43) and then are copied inside the `nginx` container with the name `nginx_key.key` and `nginx_cert.crt` as you can see [here](https://github.com/open-education-polito/fare-platform/pull/181/files/0efdfd99829943c1cfd11ccafe41185713bb776b#diff-49fb4e4cf11d540d745e3a97f5834ac763c8c6ebc0a4a7b1d9038d5ede8dfb0fR13) so inside the container the file names will be always the same. If you change the certificates you must update only the `.env` file, there is no need to change the file names also in the `default.conf` file